django/utils/html.py

```diff
@@ -218,6 +218,8 @@ def unquote_quote(segment):
     except UnicodeError:  # invalid domain part
         return unquote_quote(url)
 
+    path = unquote_quote(path)
+
```

Comment:
Why move this line? There's no behaviour change no? I think revert this please.

Reply:
Possibly, but it makes the commit here less clear. Please do revert. (We could assess whether there's a readability improvement as a separate change, but it's probably not worth it for me.)